sql/dj_setup_database.in

```diff
@@ -386,6 +397,41 @@ dump)
 	exit 0
 	;;
 
+load)
+	choice=""
+	DUMPNAME="$2"
+	if [ -z "$DUMPNAME" ]; then
+		set -- $(ls $DATABASEDUMPDIR)
```

Comment:
This whole loop can be simplified to something like:
```
    files=("$DATABASEDUMPDIR"/*)
    PS3="Which database should be loaded? "
    select choice in "${files[@]##*/}"; do
        if [ -n "$choice" ]; then
            break
        else
            echo "Invalid selection. Please try again."
        fi
    done
```

Reply:
Are arrays valid posix?

Reply:
Indeed, it's not: https://www.shellcheck.net/wiki/SC3008

Well, at some point (outside of this PR) we should have a discussion what POSIX compliance buys us